ctapipe/tools/muon_reconstruction.py

```diff
@@ -98,7 +98,7 @@ def setup(self):
                 "Outputfile {self.output} already exists, use `--overwrite` to overwrite"
             )
 
-        self.source = EventSource(parent=self)
+        self.source = EventSource.from_config(parent=self)
```

Comment:
Why this change? The current version should work, there should be no need for `from_config` or `from_url` anymore anywhere.